dev/src/write-batch.ts

```diff
@@ -565,11 +583,11 @@ export class WriteBatch {
  * Validates that the update data does not contain any ambiguous field
  * definitions (such as 'a.b' and 'a').
  *
- * @param {Map.<FieldPath, *>} data - An update map with field/value pairs.
- * @returns {boolean} 'true' if the input is a valid update map.
+ * @param data - An update map with field/value pairs.
```

Comment:
Is the `-` supposed to be there?

Reply:
Sometimes. Sometimes not :/ Removed here.